### PR TITLE
Implemented: service to sync Shopify credentials with OMS (#11)

### PR DESCRIPTION
**Pull request by adityasharma7**

### CODE REVIEW

File: service/co/hotwax/shopify/app/ShopifyAppServices.xml [hunk 1]
```diff
@@ -312,6 +312,9 @@ under the License.
             <parameter name="instanceAddress" required="true"/>
             <parameter name="instanceToken" required="true"/>
         </in-parameters>
+        <out-parameters>
+            <parameter name="error" required="false"/>
```
Comment (dixitdeepak) [inline service/co/hotwax/shopify/app/ShopifyAppServices.xml:316]: why this is needed?
moqui support rest response code, do we still need this?

File: service/co/hotwax/shopify/app/ShopifyAppServices.xml [hunk 2]
```diff
@@ -337,6 +340,57 @@ under the License.
...
+            </if>
+            <script><![CDATA[
+                import org.moqui.util.RestClient
+                import org.moqui.util.RestClient.Method
+
+                RestClient restClient = ec.service.rest()
+                restClient.method(Method.POST)
+
+                shop = shop.replace(".myshopify.com", "")
+                shopifyConfigId = shopifyConfigName + "_CONFIG"
```
Comment (dixitdeepak) [inline service/co/hotwax/shopify/app/ShopifyAppServices.xml:372]: shopifyConfigName is not defined

Comment (adityasharma7) [reply service/co/hotwax/shopify/app/ShopifyAppServices.xml]: Fixed it

File: service/co/hotwax/shopify/app/ShopifyAppServices.xml [hunk 3]
```diff
@@ -337,6 +340,57 @@ under the License.
...
+                restClient.uri("${instanceAddress}/api/service/setupShopifyShop")
+                restClient.addHeader("Authorization", "Bearer " + instanceToken);
+                restClient.addBodyParameters(parameters);
+                try {
+                    RestClient.RestResponse restResponse = restClient.call()
+                    response = restResponse.jsonObject()
+                    statusCode = restResponse.getStatusCode()
+                    if (restResponse.statusCode < 200 || restResponse.statusCode >= 300) {
+                        error = "Error connecting to OMS status code: ${statusCode} and response: ${response}"
+                        ec.logger.warn(error)
```
[the remark this answers is not in the document]
Comment (adityasharma7) [reply service/co/hotwax/shopify/app/ShopifyAppServices.xml]: Fixed it

File: service/co/hotwax/shopify/app/ShopifyAppServices.xml (hunk 1 above)
Comment (adityasharma7) [reply service/co/hotwax/shopify/app/ShopifyAppServices.xml]: Makes sense Sir. Updates the code. 
Initially, we considered that if the values are successfully stored at the keychain, it should be considered as successful irrespective of the API failure (which could be due to certain reasons like token expiration, instance down, etc), and if there is an error message it should be displayed.

**Pull request merged**